config.json

```diff
@@ -1,6 +1,6 @@
 {
 	"admin_server" : {
-		"listen_url" : "127.0.0.1:3333",
+		"listen_url" : "0.0.0.0:3333",
```

Comment:
This should not be changed in this pull request - the default admin will need to stay at 127.0.0.1